osm2pgsql.lua

```diff
@@ -499,128 +499,116 @@ function osm2pgsql.process_node(object)
         object.tags.wikidata == 'Q994730') then
         output_hstore['disputed_by'] = 'CN;RU;IN;GR'
     end
--- Recast Northern Cyprus as country label and turn off for several POVs including China and Russia
+
+-- Turn off Northern Cyprus label
     if object.tags.place and object.tags.wikidata == 'Q23681' then
-        output_hstore['place'] = 'country'
-        output_hstore['disputed_by'] = 'CN;RU;IN;GR;CY'
+        output_hstore['place'] = nil
```

Comment:
Instead, we still want to set this to place = country... and then (once it's removed from the NE join table) the kind will calc to country, but failing the join it'll fallback to `unrecognized` by default.